rebasehelper/cli.py

```diff
@@ -123,6 +124,12 @@ def run():
             else:
                 main_handler.setLevel(logging.DEBUG)
             ConsoleHelper.use_colors = ConsoleHelper.should_use_colors(config)
+            if config.bugzilla_id:
+                repo_path, config.sources = BugzillaHelper.prepare_rebase_repository(config.bugzilla_id)
+                try:
+                    os.chdir(repo_path)
+                except OSError:
+                    raise RebaseHelperError("Could not change directory into the cloned repository")
```

Comment:
I think `to` instead of `into` is more suitable in this context. According to google search, `to` is used about 7 times more often than `into` ;-)
Also, why double quotes? There is no reason to use them here. We should finally agree on a convention where to use which quotes, but that's for another discussion.

Reply:
Good point with the quotes. Our code is inconsistent in this regard and we should change that. Perhaps using `black` or a similar tool could be a good solution? If I remember correctly, `black` uses double quotes for all strings, though I am not very keen on some aspects of the code formatting that `black` produces.

Reply:
I'm not inclined to using black. My preference is to use single quotes everywhere and leave double quotes for docstrings. There is also another related question - should we quote paths, names etc. in log messages and exceptions? I think in most cases it's unnecessary, but it would be nice to be consistent in this.